frontend/app/services/letters-service.server.ts

```diff
@@ -0,0 +1,46 @@
+import { z } from 'zod';
+
+import { getEnv } from '~/utils/env.server';
+import { getLogger } from '~/utils/logging.server';
+
+const letterSchema = z.object({
+  dateSent: z.date().optional(),
+  letterTypeCode: z.string().optional(),
+  referenceId: z.string().uuid().optional(),
+});
+
+export type LettersInfo = z.infer<typeof letterSchema>;
+
+function createLettersService() {
+  const logger = getLogger('letters-service.server');
+  const { INTEROP_API_BASE_URI } = getEnv();
+
+  /**
+   *
+   * @param { userId }
+   * @returns returns the letters based off  @param userId
+   */
+  async function getLetters(requestBody: { userId: string }) {
+    const url = `${INTEROP_API_BASE_URI}/letters?userId=${requestBody.userId}`;
```

Comment:
```suggestion
    const url = new URL(`${INTEROP_API_BASE_URI}/letters`);
    url.searchParams.set('userId', requestBody.userId);
```

Reply:
Since the other services use the former format, I want to keep it consistent. Otherwise, I feel it should be changed all at the same time.

Reply:
Other services doens't have `searchParams` change this one and we can refactor the other places in another PR.

Reply:
That's fair. I'll resolve the comment.